src/main/java/team/gif/robot/subsystems/TelescopingArm.java

```diff
@@ -0,0 +1,24 @@
+package team.gif.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import team.gif.robot.RobotMap;
+
+public class TelescopingArm extends SubsystemBase {
+    public static CANSparkMax telescopingMotor;
+
+    public TelescopingArm() {
+        telescopingMotor = new CANSparkMax(RobotMap.TELESCOPING_MOTOR, CANSparkMaxLowLevel.MotorType.kBrushless);
+        telescopingMotor.setInverted(true);
+        telescopingMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
+    }
+
+    public void setTelescopingMotor(double percent) {
+        telescopingMotor.set(percent);
+    }
+
```

Comment:
In game time conditions, we are probably not going to use the manual control of the arm. We will just use something like:
ArmOut()
ArmIn()

We'll call these depending on which location we are going to.
We keep the manual control though in case the aux needs it for some unforeseen reason.

Go ahead and create these functions for now. I think the SparkMax uses a relative sensor position, so ArmIn will be a command which runs until the position = 0, and ArmOut will be a command which runs until the position is MAX_OUT_POSITION

Reply:
Are encoders on the telescoping arm a Talon?